src/structurizr/model/model_item.py

```diff
@@ -47,10 +48,10 @@ class ModelItemIO(BaseModel, ABC):
     perspectives: List[PerspectiveIO] = Field(default=())
 
     @validator("tags", pre=True)
-    def split_tags(cls, tags: Union[str, List[str]]) -> List[str]:
+    def split_tags(cls, tags: Union[str, List[str], Set[str]]) -> List[str]:
```

Comment:
I think there's no need to be strict with the types here.

```suggestion
    def split_tags(cls, tags: Union[str, Iterable[str]]) -> List[str]:
```

Reply:
Good point.  Done.